{"uid":"5eca256d977769f7","name":"Login with valid credentials test","fullName":"tests.test_login.TestLogin#test_valid_login","historyId":"e7f254b9bfd5f2bc8d4a1ffa6dc7aebb","time":{"start":1701478879701,"stop":1701478889931,"duration":10230},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5628625395e3 <unknown>\n#1 0x5628621fc0b7 <unknown>\n#2 0x562862249f53 <unknown>\n#3 0x56286224a051 <unknown>\n#4 0x56286228f9c4 <unknown>\n#5 0x562862270f1d <unknown>\n#6 0x56286228cb3d <unknown>\n#7 0x562862270cc3 <unknown>\n#8 0x56286223c0e4 <unknown>\n#9 0x56286223d0ae <unknown>\n#10 0x5628624ffce1 <unknown>\n#11 0x562862503b7e <unknown>\n#12 0x5628624ed4b5 <unknown>\n#13 0x5628625047d6 <unknown>\n#14 0x5628624d0dbf <unknown>\n#15 0x562862527748 <unknown>\n#16 0x562862527917 <unknown>\n#17 0x562862538773 <unknown>\n#18 0x7f3138494ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f5660c2c490>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"1bd867f5d57af5f0551ae28f69684663\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f5660b40220>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5628625395e3 <unknown>\nE       #1 0x5628621fc0b7 <unknown>\nE       #2 0x562862249f53 <unknown>\nE       #3 0x56286224a051 <unknown>\nE       #4 0x56286228f9c4 <unknown>\nE       #5 0x562862270f1d <unknown>\nE       #6 0x56286228cb3d <unknown>\nE       #7 0x562862270cc3 <unknown>\nE       #8 0x56286223c0e4 <unknown>\nE       #9 0x56286223d0ae <unknown>\nE       #10 0x5628624ffce1 <unknown>\nE       #11 0x562862503b7e <unknown>\nE       #12 0x5628624ed4b5 <unknown>\nE       #13 0x5628625047d6 <unknown>\nE       #14 0x5628624d0dbf <unknown>\nE       #15 0x562862527748 <unknown>\nE       #16 0x562862527917 <unknown>\nE       #17 0x562862538773 <unknown>\nE       #18 0x7f3138494ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1701478879700,"stop":1701478879700,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1701478879700,"stop":1701478879700,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"secret_data","time":{"start":1701478879701,"stop":1701478879701,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"json_data","time":{"start":1701478879701,"stop":1701478879701,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5628625395e3 <unknown>\n#1 0x5628621fc0b7 <unknown>\n#2 0x562862249f53 <unknown>\n#3 0x56286224a051 <unknown>\n#4 0x56286228f9c4 <unknown>\n#5 0x562862270f1d <unknown>\n#6 0x56286228cb3d <unknown>\n#7 0x562862270cc3 <unknown>\n#8 0x56286223c0e4 <unknown>\n#9 0x56286223d0ae <unknown>\n#10 0x5628624ffce1 <unknown>\n#11 0x562862503b7e <unknown>\n#12 0x5628624ed4b5 <unknown>\n#13 0x5628625047d6 <unknown>\n#14 0x5628624d0dbf <unknown>\n#15 0x562862527748 <unknown>\n#16 0x562862527917 <unknown>\n#17 0x562862538773 <unknown>\n#18 0x7f3138494ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f5660c2c490>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"1bd867f5d57af5f0551ae28f69684663\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f5660b40220>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5628625395e3 <unknown>\nE       #1 0x5628621fc0b7 <unknown>\nE       #2 0x562862249f53 <unknown>\nE       #3 0x56286224a051 <unknown>\nE       #4 0x56286228f9c4 <unknown>\nE       #5 0x562862270f1d <unknown>\nE       #6 0x56286228cb3d <unknown>\nE       #7 0x562862270cc3 <unknown>\nE       #8 0x56286223c0e4 <unknown>\nE       #9 0x56286223d0ae <unknown>\nE       #10 0x5628624ffce1 <unknown>\nE       #11 0x562862503b7e <unknown>\nE       #12 0x5628624ed4b5 <unknown>\nE       #13 0x5628625047d6 <unknown>\nE       #14 0x5628624d0dbf <unknown>\nE       #15 0x562862527748 <unknown>\nE       #16 0x562862527917 <unknown>\nE       #17 0x562862538773 <unknown>\nE       #18 0x7f3138494ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1701478879702,"stop":1701478889930,"duration":10228},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5628625395e3 <unknown>\n#1 0x5628621fc0b7 <unknown>\n#2 0x562862249f53 <unknown>\n#3 0x56286224a051 <unknown>\n#4 0x56286228f9c4 <unknown>\n#5 0x562862270f1d <unknown>\n#6 0x56286228cb3d <unknown>\n#7 0x562862270cc3 <unknown>\n#8 0x56286223c0e4 <unknown>\n#9 0x56286223d0ae <unknown>\n#10 0x5628624ffce1 <unknown>\n#11 0x562862503b7e <unknown>\n#12 0x5628624ed4b5 <unknown>\n#13 0x5628625047d6 <unknown>\n#14 0x5628624d0dbf <unknown>\n#15 0x562862527748 <unknown>\n#16 0x562862527917 <unknown>\n#17 0x562862538773 <unknown>\n#18 0x7f3138494ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[{"uid":"5fe0bdf98768524f","name":"public ip address","source":"5fe0bdf98768524f.txt","type":"text/plain","size":13},{"uid":"858ed3160596b108","name":"Cookies","source":"858ed3160596b108.json","type":"application/json","size":2},{"uid":"d091c3f4777a0f02","name":"Session Storage","source":"d091c3f4777a0f02.json","type":"application/json","size":2},{"uid":"1fbdd32bfac560fd","name":"Local Storage","source":"1fbdd32bfac560fd.json","type":"application/json","size":2},{"uid":"7deba5ecbf6bc818","name":"Console Logs","source":"7deba5ecbf6bc818.json","type":"application/json","size":2},{"uid":"bef73264dc3e7d1","name":"Full Page Screenshot","source":"bef73264dc3e7d1.png","type":"image/png","size":2669394},{"uid":"b73a2a41983681b3","name":"URL","source":"b73a2a41983681b3.uri","type":"text/uri-list","size":25},{"uid":"db106fdc5c31bde3","name":"Network Logs","source":"db106fdc5c31bde3.json","type":"application/json","size":14025}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true},"afterStages":[],"labels":[{"name":"tag","value":"Tagged test"},{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"severity","value":"blocker"},{"name":"tag","value":"run(order=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az841-991"},{"name":"thread","value":"2257-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","run(order=1)"]},"source":"5eca256d977769f7.json","parameterValues":[]}